android/src/main/java/com/adpdigital/push/rn/AdpPushClientModule.java

```diff
@@ -1,1132 +1,1213 @@
-package com.adpdigital.push.rn;

```

Comment:
This file changes seems strange @Husseinhj Please check and let @Mr-Hqq to update the PR if needed